### PR TITLE
Refactor Build container images page

### PR DESCRIPTION
**Pull request by nginx-seanmoloney**
### Proposed changes
Review the [docker-images](https://docs.nginx.com/nginx-agent/installation-upgrade/container-environments/docker-images/) page on the official documentation. Add link to the official docker images. Fix issues with existing commands. 

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [ ] I have run ```make install-tools``` and have attached any dependency changes to this pull request
- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works
- [ ] If applicable, I have checked that any relevant tests pass after adding my changes
- [ ] If applicable, I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))
- [ ] If applicable, I have tested my cross-platform changes on Ubuntu 22, Redhat 8, SUSE 15 and FreeBSD 13

### CODE REVIEW

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 1]
```diff
@@ -215,9 +227,10 @@ You can pass the following arguments when running the **make** command to build
 {{<bootstrap-table "table table-striped table-border">}}
```
Comment (dhurley) [inline site/content/installation-upgrade/container-environments/docker-images.md:227]: One line 222 & 223 it should be:
```
- `make oss-image` builds an image containing NGINX Agent and NGINX open source.
- `make image` builds an image containing NGINX Agent and NGINX Plus.
```

**Review by dhurley — COMMENTED** [review]
I think the `Build the official NGINX Agent image with Docker` section should be removed and replaced with how to pull the official NGINX Agent images.

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 2]
```diff
@@ -17,6 +17,13 @@ If you want to use NGINX Agent with NGINX Plus, you need to purchase an NGINX Pl
 
 See the requirements and supported operating systems in the [NGINX Agent Technical Specifications]({{< relref "technical-specifications.md" >}}) topic.
 
+## Deploying NGINX and NGINX Plus on Docker
+
+You can access container images by visiting the following page: 
+
+[Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/)
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:24]:
```suggestion
## Deploy NGINX and NGINX Plus on Docker

Docker images are available in the [Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/) NGINX documentation. 
```

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 3]
```diff
@@ -17,6 +17,13 @@ If you want to use NGINX Agent with NGINX Plus, you need to purchase an NGINX Pl
 
 See the requirements and supported operating systems in the [NGINX Agent Technical Specifications]({{< relref "technical-specifications.md" >}}) topic.
 
+## Deploying NGINX and NGINX Plus on Docker
+
+You can access container images by visiting the following page: 
+
+[Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/) 
+
+This page provides instructions on how to build images with the NGINX agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:26]:
```suggestion
The document provides instructions on how to build images with NGINX Agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.
```

Product names without ~the~ and capitalized.

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 4]
```diff
@@ -199,6 +206,11 @@ docker run --name nginx-agent -d \
   nginx-agent
 ```
 
+To ensure that the REST Interface is correctly configured, you can curl the following endpoint from your terminal:
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:209]:
```suggestion
To ensure that the REST Interface is correctly configured, you can use the `curl` command targeting the following endpoint from your terminal:

```

Leave a blank space between paragraphs and code blocks.

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 5]
```diff
@@ -199,6 +206,11 @@ docker run --name nginx-agent -d \
...
+```shell
+curl 0.0.0.0:8038/nginx/
+```
+
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:213]: Should we also provide an example output or what to expect from this command to confirm the test?

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 6]
```diff
@@ -230,75 +243,18 @@ Keep the following information in mind when using the NGINX Agent [Dockerfiles](
...
-PACKAGE_NAME=[PATH-TO-PACKAGE] make official-oss-image
+IMAGE_BUILD_TARGET=install-agent-repo make oss-image
 ```
 
-### Example build commands by distribution
-
-{{<tabs name="build-oss-image">}}
-
-{{%tab name="alma linux"%}}
+To build an image with debian and older version of the NGINX Agent you can run the following command.
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:252]:
```suggestion
To build an image with Debian and an older version of NGINX Agent you can run the following command:
```

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 7]
```diff
@@ -230,75 +243,18 @@ Keep the following information in mind when using the NGINX Agent [Dockerfiles](
 
 ### Build NGINX open source images
 
-Run the following **make** command to build the default image, which uses Alpine 3.19 as the base image.
+Run the following **make** command to build the default image, which uses Alpine as the base image.
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:246]:
```suggestion
Run the following **make** command to build the default image, which uses Alpine as the base image:
```

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 8]
```diff
@@ -307,75 +263,12 @@ PACKAGE_NAME=[PATH-TO-PACKAGE] make oss-image OS_RELEASE=ubuntu
...
-{{%tab name="suse"%}}
-
-```shell
-PACKAGE_NAME=[PATH-TO-PACKAGE] OS_RELEASE=suse OS_VERSION=sle15 make official-plus-image
-```
-
-{{% /tab %}}
-
-{{%tab name="ubuntu"%}}
+To build an image with debian and older version of the NGINX Agent you can run the following command.
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:269]:
```suggestion
To build an image with Debian and an older version of NGINX Agent you can run the following command:
```

File: site/content/installation-upgrade/container-environments/docker-images.md (hunk 1 above)
Comment (Jcahilltorre) [reply site/content/installation-upgrade/container-environments/docker-images.md:226]: updated

**Review by Jcahilltorre — APPROVED** [review]

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 9]
```diff
@@ -17,6 +17,11 @@ If you want to use NGINX Agent with NGINX Plus, you need to purchase an NGINX Pl
 
 See the requirements and supported operating systems in the [NGINX Agent Technical Specifications]({{< relref "technical-specifications.md" >}}) topic.
 
+## Deploy Offical NGINX and NGINX Plus Containers
+
+Docker images are available in the [Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/) NGINX documentation.
+
+The document provides instructions on how to build images with NGINX Agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:24]:
```suggestion
This guide provides instructions on how to build images with NGINX Agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.
```

File: site/content/installation-upgrade/container-environments/docker-images.md [hunk 10]
```diff
@@ -105,61 +110,17 @@ In order to build a container image with NGINX Plus, you must provide the SSL ce
...
 
-## Build the official NGINX Agent image with Docker
-
-{{<tabs name="build-image">}}
-
-{{%tab name="NGINX Open Source"%}}
+## Run the NGINX Agent container
 
-Change to the directory where the Dockerfile is located:
+Here is an example of how to run the NGINX Agent container using Docker:
```
Comment (Jcahilltorre) [inline site/content/installation-upgrade/container-environments/docker-images.md:115]:
```suggestion
To run NGINX Agent container using Docker use the following command:
```

**Review by Jcahilltorre — APPROVED** [review]

**Comment by Jcahilltorre** [comment]
I have hereby read the F5 CLA and agree to its terms

**Review by aphralG — APPROVED** [review]

**Comment by oCHRISo** [comment]
This all looks good. One thing for the "Run the NGINX Agent container" section. Should we give another command so an end user knows how to mount an nginx-agent.conf file? 

When running `docker run --name nginx-agent -d nginx-agent` Agent has default config
Could provide this command to mount a user's nginx-agent.conf file
```docker run --name nginx-agent -d \
  -v /path/to/your/nginx-agent.conf:/etc/nginx-agent/nginx-agent.conf \
  nginx-agent
```

**Review by dhurley — APPROVED** [review]

**Pull request closed without merging**